src/openzaak/components/documenten/models.py

```diff
@@ -752,6 +756,8 @@ class ObjectInformatieObject(models.Model, CMISClientMixin):
         fk_field="_besluit", url_field="_besluit_url", blank=True, null=True,
     )
 
+    verzoek = models.URLField(_("extern verzoek"), blank=True, max_length=1000)
```

Comment:
I think we can clean up this entire thing by only having one `_object_url` field - we can't guarantee referential integrity on that anyway. The `object_type` then determines how to interpret that URL, and we can simplify the DB constraints substantially and probably get rid of a lot of code.